src/lib/oper_data.c

```diff
@@ -0,0 +1,95 @@
+/* SPDX-License-Identifier: AGPL-3.0-or-later */
+/*
+ * Authors:     Amjad Daraiseh, adaraiseh@okdanetworks.com>
+ *
+ *              This program is free software; you can redistribute it and/or
+ *              modify it under the terms of the GNU Affero General Public
+ *              License Version 3.0 as published by the Free Software Foundation;
+ *              either version 3.0 of the License, or (at your option) any later
+ *              version.
+ *
+ * Copyright (C) 2024 Okda Networks, <contact@okdanetworks.com>
+ */
+
+#include "oper_data.h"
+#include "cmdgen.h"
+
+/**
+ * Struct store yang modules names and their iproute2 show cmd start.
+ */
+struct module_sh_cmdstart {
+    const char *module_name; // Stores Module name
+    const char *showcmd_start; // Store iproute2 show command related to the module
+} ipr2_sh_cmdstart[] = { { "iproute2-ip-link", "ip link show" },
```

Comment:
this is fine, but we can use yang extensions to specify the cmd, and the Keys that can be used by iproute2 command.

Reply:
yes we might use the extensions later, after we add the operational data yang.